src/main/java/foxtrot/uniform/charlie/kilo/model/MovieSession.java

```diff
@@ -20,44 +20,41 @@
     @JoinColumn(name = "movie_id")
     private Movie movie;
     @ManyToOne (fetch = FetchType.LAZY)
-    @JoinColumn(name = "cinemaHall_id")
+    @JoinColumn(name = "cinema_hall_id")
     private CinemaHall cinemaHall;
+    @JoinColumn(name = "show_time")
```

Comment:
just a remark: we should better use `@Column` here - not `@JoinColumn` because we don't have relationship with the field here